lib/rspec/matchers/built_in.rb

```diff
@@ -12,6 +12,7 @@ module Matchers
     module BuiltIn
       autoload :BeAKindOf,               'rspec/matchers/built_in/be_kind_of'
       autoload :BeAnInstanceOf,          'rspec/matchers/built_in/be_instance_of'
+      autoload :BeASubclassOf,           'rspec/matchers/built_in/be_subclass_of'
```

Comment:
I was attempting to match the implementation of `be_a_kind_of` (primary method), which is backed by a matcher class called `BeAKindOf` in a file named `be_kind_of.rb`.

I will happily change to make the method and file name the same. Should the same be done for `BeAKindOf` (currently in `be_kind_of.rb`) and `BeAnInstanceOf` (currently in `be_instance_of.rb`) also?

Note that, as far as I can tell, the matcher class names contain the `A` and `An` for the purpose of automatically generating the matcher descriptions rather than anything else.